pkg/performanceprofile/cmd/render/render.go

```diff
@@ -161,6 +159,9 @@ func render(inputDir, outputDir string) error {
 		}
 	}
 
+	// Append any missing default manifests (i.e. `master`/`worker`)
+	mcPools = util.AppendMissingDefaultMCPManifests(mcPools)
```

Comment:
Could it be possible to add this functionality to 
https://github.com/openshift/cluster-node-tuning-operator/blob/master/pkg/tuned/cmd/render/render.go
?

Reply:
That's a good point, the logic seems the same, @jlojosnegros any concerns with adding this call to the tuned renderer?

Reply:
Confirmed in Slack, added to tuned renderer